Find a functionally equivalent code:android.graphics.Picture.getHeight:COMMENT
<android.graphics.Picture: int getHeight()>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Get the height of the picture as passed to beginRecording. This
 * does not reflect (per se) the content of the picture.
 */

@@ -1 +1,3 @@
-
+{
+    return nativeGetHeight(mNativePicture);
+}

